common/djangoapps/util/tests/test_password_policy_validators.py

```diff
@@ -63,8 +63,15 @@ def test_unicode_password(self):
 
     def test_password_unicode_normalization(self):
         """ Tests that validate_password normalizes passwords """
-        #TODO @Dillon will do this test since it relates to validation.
-        pass
+        # s ̣ ̇ (s with combining dot below and combining dot above)
+        not_normalized_password = u'\u0073\u0323\u0307'
+        self.assertEqual(len(not_normalized_password), 3)
+
+        # When we normalize we expect the not_normalized password to fail
+        # becuase it should be normalized to u'\u1E69' -> ṩ
+        with PASSWORD_UNICODE_NORMALIZE_FLAG.override(active=True):
+            self.validation_errors_checker(not_normalized_password,
+                                           'This password is too short. It must contain at least 2 characters.')
```

Comment:
I might also throw in a call without the flag set - just to confirm that the validation is checking the waffle flag.